### ISSUE TITLE
`excessive-visibility` lint doesn't flag unused public methods

### ISSUE THREAD

**Issue by yegor256** [objectionary/lints#1207]
Consider this object:

```
[t] > phrase
  pos.gte 0 > multi-words
  t.index-of " " > pos

  (book "Object Thinking").multi-words ++> can-find-space-in-name
```

Here, the `pos` is not used by any tests. It can and should be made private:

```
[t] > phrase
  pos.gte 0 > multi-words
  t.index-of " " >> pos
                ^^^^
                here
```

Let's create a lint that would catch this. If no tests refer to a method, it must be private/obfuscated (`>>`).

**Comment by aryansk** [comment]
I’m taking the narrow excessive-visibility lint proposal and adding focused XSL fixture coverage for a public method that no unit test references, plus the allowed control case.
